benchmarks/pipe.py

```diff
@@ -245,10 +253,12 @@ def train(data_config, model, benchmark_config, args):
     if pipe_group and pipe_group.rank() != 0 and pipe_group.rank() != (pipe_group.size() - 1):
         lm_dataloader = get_fake_dataloader()
 
+    total_words = 0
     for i, batch in enumerate(lm_dataloader):
 
         if args.max_batch and i > args.max_batch:
             break
+        total_words += np.prod(list(batch["input"].size()))
```

Comment:
batch["input"].numel() returns the number of elements, which is that what this is computing I think.

For total_words, we usually compute it exclude the padding tokens. here, I don't know the data contains padding or not. either way, using total_tokens might be more accurate.